app/components/primer/alpha/tab_panels.rb

```diff
@@ -19,7 +19,7 @@ class TabPanels < Primer::Component
       # @param id [String] Unique ID of tab.
       # @param selected [Boolean] Whether the tab is selected.
       # @param system_arguments [Hash] <%= link_to_system_arguments_docs %>
-      renders_many :tabs, lambda { |id:, selected: false, **system_arguments|
+      renders_many :tabs, lambda { |id: self.class.generate_id, selected: false, **system_arguments|
```

Comment:
I updated this to auto generate one if it's not passed in. Didn't want to have to enforce making up an id for every selectpanel